### PR TITLE
services/horizon: Change protocol version check error

### PR DESCRIPTION
**Pull request by bartekn**
Changed `Protocol version not supported` to `Error checking protocol version is supported` message because an error can be unrelated to protocol version mismatch (ex. error connecting to ledger backend).

### CODE REVIEW

File: services/horizon/internal/ingest/processor_runner.go [hunk 1]
```diff
@@ -206,7 +206,7 @@ func (s *ProcessorRunner) RunHistoryArchiveIngestion(checkpointLedger uint32) (i
 		}
 	} else {
 		if err = s.checkIfProtocolVersionSupported(checkpointLedger); err != nil {
-			return changeStats.GetResults(), errors.Wrap(err, "Protocol version not supported")
+			return changeStats.GetResults(), errors.Wrap(err, "Error checking protocol version is supported")
```
Comment (ire-and-curses) [inline services/horizon/internal/ingest/processor_runner.go:209]: The change makes sense, but maybe we can word it more clearly?
```suggestion
Error while checking for supported protocol version (<= 15)
```
Where the reported number is a variable

**Review by abuiles — APPROVED** [review]

File: services/horizon/internal/ingest/processor_runner.go [hunk 2]
```diff
@@ -279,7 +279,7 @@ func (s *ProcessorRunner) RunTransactionProcessorsOnLedger(ledger uint32) (io.St
 	}
 
 	if err = s.checkIfProtocolVersionSupported(ledger); err != nil {
-		return ledgerTransactionStats.GetResults(), errors.Wrap(err, "Protocol version not supported")
+		return ledgerTransactionStats.GetResults(), errors.Wrap(err, "Error checking protocol version is supported")
```
Comment (2opremio) [inline services/horizon/internal/ingest/processor_runner.go:282]:
```suggestion
		return ledgerTransactionStats.GetResults(), errors.Wrap(err, "Error checking whether protocol version is supported")
```

Comment (2opremio) [reply services/horizon/internal/ingest/processor_runner.go:282]: And the same for the rest

**Comment by 2opremio** [comment]
Alternative suggestion for the text. Otherwise LGTM

**Review by 2opremio — APPROVED** [review]

**Comment by 2opremio** [comment]
@bartekn can you also correct the other instances of the text?

**Comment by 2opremio** [comment]
In fact, it may be worth factoring it out in a constant (or error function)

**Pull request merged**